e2e/rbd.go

```diff
@@ -791,6 +791,112 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("test RBD volume encryption with user secrets based SecretsMetadataKMS", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass with error %v", err)
+				}
+				scOpts := map[string]string{
+					"encrypted":       "true",
+					"encryptionKMSID": "user-ns-secrets-metadata-test",
+				}
+				err = createRBDStorageClass(f.ClientSet, f, defaultSCName, nil, scOpts, deletePolicy)
+				if err != nil {
+					e2elog.Failf("failed to create storageclass with error %v", err)
+				}
+
+				// user provided namespace where secret will be created
+				namespace := rookNamespace
```

Comment:
as we don't create anything in the rook namespace, instead of rookNamespace it should be cephcsiNamespace?

Reply:
The aim of this particular test case is to test if secret can be fetched from user specified `secretName` & `secretNamespace`. 
( user specified `secretName` & tenant/k8s namespace of PVC is tested in the other e2e)

`cephcsiNamespace == default` but default is replaced with e2e testing unique namespace in `kms-config.yaml`, that's why I used `rookNamespace` [where secret will be created(then destroyed) in the same function being referred for testing] .

Reply:
Consider using a new namespace `tenant` as example. I might do that for #2246 too.

Reply:
ok, I'll add a new namespace `tenant` , sounds good :+1: